app/components/shared/gcse_qualification_cards_component.html.erb

```diff
@@ -46,10 +46,22 @@
               <dt class="app-qualification__key govuk-visually-hidden">Awarded</dt>
               <dd class="app-qualification__value app-qualification__value--caption">
```

Comment:
Whilst we’re here could we remove this 'caption' class, so that the text appears in regular black like the rest rather than very very dark grey?

```suggestion
              <dd class="app-qualification__value">
```